src/locales/pt_BR/battle.json

```
@@ -32,6 +32,7 @@
   "attackMissed": "{{pokemonNameWithAffix}} desviou do ataque!",
   "attackHitsCount": "Acertou {{count}} vezes.",
   "rewardGain": "Você recebeu\n{{modifierName}}!",
+  "rewardGainCount": "Você recebeu\n{{count}} {{modifierName}}!",
```

```suggestion
  "rewardGainCount": "Você recebeu\n{{count}} $t(modifierName, {\"count\": {{count}} })!",
```

For possible plural handling